### PR TITLE
[breadcrumb] Fix unknown projects

### PR DESCRIPTION
**Pull request by ericdallo**
If user opens a file from an unknown project, we should not fail.

image

### CODE REVIEW

**Comment by ericdallo** [comment]
Force merging this as it is wrong on master

File: lsp-mode.el [hunk 1]
```diff
@@ -2203,17 +2208,18 @@ PATH is the current folder to be checked."
...
-                  (format "%s%s %s"
-                          (if last-dirs
-                              (concat last-dirs " ")
-                            (if (eq (cl-first lsp-headerline-breadcrumb-segments) 'path-up-to-project)
-                                ""
-                              " "))
-                          (lsp--headerline-breadcrumb-arrow-icon)
-                          (propertize next-dir 'font-lock-face lsp-headerline-breadcrumb-prefix-face)))
-                (lsp--headerline-path-up-to-project-root (lsp-workspace-root) (f-parent (buffer-file-name))) nil)))
+  (when (and (member 'path-up-to-project lsp-headerline-breadcrumb-segments))
```
Comment (leungbk) [inline lsp-mode.el:2211]: The `and` shouldn't be required?

Comment (ericdallo) [reply lsp-mode.el:2211]: Yeah, I'll remove this, thanks!

Comment (ericdallo) [reply lsp-mode.el:2211]: Done here: https://github.com/emacs-lsp/lsp-mode/pull/1913

**Pull request merged**